clang/unittests/Format/FormatTest.cpp

```diff
@@ -25831,19 +25843,108 @@ TEST_F(FormatTest, RequiresClausesPositions) {
 
   verifyFormat("template <typename T>\n"
                "int S::bar(T t) &&\n"
-               "requires F<T> {\n"
+               "requires F<T>\n"
+               "{\n"
                "  return 5;\n"
                "}",
                Style);
 
   verifyFormat("template <typename T>\n"
                "int bar(T t)\n"
-               "requires F<T> {\n"
+               "requires F<T>\n"
+               "{\n"
                "  return 5;\n"
                "}",
                Style);
 
+  Style.RequiresClausePosition = FormatStyle::RCPS_OwnLineWithBrace;
+  Style.IndentRequiresClause = true;
+
+  // The default in LLVM style is REI_OuterScope, but these tests were written
+  // when the default was REI_Keyword.
```

Comment:
Why?

Sure you can test the combination, but you always have to test LLVM + that single option.